composer.json

```diff
@@ -169,17 +181,17 @@
                 "[web-root]/web.config": {
                     "mode": "skip"
                 }
-            },
-            "initial": {
-                "sites/default/default.services.yml": "[web-root]/sites/default/services.yml",
-                "sites/default/default.settings.php": "[web-root]/sites/default/settings.php"
             }
         }
     },
     "scripts": {
         "post-install-cmd": [
             "@prepare"
         ],
+        "post-drupal-scaffold-cmd": [
```

Comment:
Please add `"cp tests/behat/behat.local.example.yml tests/behat/behat.local.yml",` to create local behat yml file.